src/components/Panel.ts

```diff
@@ -257,7 +261,7 @@ class Panel implements FlickingPanel {
   }
 
   public getElement(): HTMLElement {
-    return this.element;
+    return this.element!;
```

Comment:
@WoodNeck What returns if `this.element` is null?

Reply:
I thought it won't happen, as `element` will be `null` only if `renderExternal` is true.
And if it is, I thought setElement() will be called immediately after resizing in `sync()`.
I also think that kind of code is unstable, do you know any bypass for it?